wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandScheduler.java

```diff
@@ -325,12 +325,9 @@ public void run() {
         schedule(subsystemCommand.getValue());
       }
     }
+    Tracer.endTrace(); // Commands
 
     m_watchdog.disable();
-    if (m_watchdog.isExpired()) {
-      System.out.println("CommandScheduler loop overrun");
-      m_watchdog.printEpochs();
-    }
```

Comment:
This should not be removed- you can remove `printEpochs`, but there should still be a notification of an overrun. It should also print a short message on how to check the Tracer outputs- teams are used to seeing the breakdown in the console.

Reply:
I elected to utilize the watchdog callback feature as the sole source of printing for loop overruns